### PR TITLE
Make resource configurable for pod/job + don't keep history forever

### PR DESCRIPTION
**Pull request by toban**
This releases 0.0.4 of wbaas-backup

Fix for https://phabricator.wikimedia.org/T307199, the pods on production are evicted because they don't specify the disk required.

Make resources configurable and request a 60GB disk at startup
Don't keep history forever.

### CODE REVIEW

**Comment by toban** [comment]
https://github.com/wmde/wbaas-deploy/pull/262/files

File: charts/wbaas-backup/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This chart does not yet follow Semantic versioning.
 
+## 0.0.4
+
+Adds `ephemeral-storage` request of 60GB by default, limits failed/successful history to 7 runs
+
```
Comment (toban) [inline charts/wbaas-backup/CHANGELOG.md:8]:
```suggestion
Sets request/limit pairs
```

File: charts/wbaas-backup/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This chart does not yet follow Semantic versioning.
 
+## 0.0.4
+
+Adds `ephemeral-storage` request of 60GB by default, limits failed/successful history to 7 runs
+Sets request/limit pairs
```
Comment (toban) [inline charts/wbaas-backup/CHANGELOG.md:8]:
```suggestion
Sets request/limit pairs

```

**Review by rosalieper — APPROVED** [review]
Looks good but I did not try it out yet.

**Pull request merged**